portability/toku_portability.h

```diff
@@ -69,7 +69,7 @@ Copyright (c) 2006, 2015, Percona and/or its affiliates. All rights reserved.
 #include <sys/stat.h>
 #include <stdio.h>
 
-#if __FreeBSD__
+#if defined(__FreeBSD__) && __FreeBSD__
```

Comment:
Just if defined or ifdef, no need to check the actual value according to https://www.freebsd.org/doc/en/books/porters-handbook/porting-versions.html